<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatwindow</class>
 <widget class="QMainWindow" name="chatwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Slick Serverless Sockets Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCommandLinkButton" name="quitButton">
      <property name="text">
       <string>Quit</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ipLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>127.0.0.1:52337</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnProfile">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/rsrc/people</normaloff>:/rsrc/people</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnClear">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/rsrc/clearicon</normaloff>:/rsrc/clearicon</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QTreeWidget" name="chatTree">
      <property name="indentation">
       <number>0</number>
      </property>
      <attribute name="headerMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>From</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Messsage</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPending">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pending Offline Messages: 0</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4">
     <widget class="QTextEdit" name="chatBox">
      <property name="maximumSize">
       <size>
        <width>100000</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="QPushButton" name="btnSend">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Send!</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/rsrc/mail</normaloff>:/rsrc/mail</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
